src/devices/RGBLedMatrix/RgbLedMatrix.cs

```diff
@@ -0,0 +1,641 @@
+            OpenAndWriteToPin(_mapping.Clock, PinValue.Low);
+            OpenAndWriteToPin(_mapping.Latch, PinValue.Low);
+
+            OpenAndWriteToPin(_mapping.R1, PinValue.Low);
+            OpenAndWriteToPin(_mapping.G1, PinValue.Low);
+            OpenAndWriteToPin(_mapping.B1, PinValue.Low);
+            OpenAndWriteToPin(_mapping.R2, PinValue.Low);
+            OpenAndWriteToPin(_mapping.G2, PinValue.Low);
+            OpenAndWriteToPin(_mapping.B2, PinValue.Low);
+
+            _rowSetMasks   = new ulong [_deviceRows >> 1];
+
+            for (int i = 1; i < _deviceRows >> 1; i++)
+            {
+                if ((i & 1)    != 0) _rowSetMasks[i] |= _gpio.AMask;
+                if ((i & 2)    != 0) _rowSetMasks[i] |= _gpio.BMask;
+                if ((i & 4)    != 0) _rowSetMasks[i] |= _gpio.CMask;
+                if ((i & 8)    != 0) _rowSetMasks[i] |= _gpio.DMask;
+                if ((i & 0x10) != 0) _rowSetMasks[i] |= _gpio.EMask;
+            }
+
+            _colorsMake = new ulong[16]; // 8 for RGB1 and 8 for RGB2
+            for (int i = 1; i < 8; i++)
+            {
+                if ((i & 1) != 0) { _colorsMake[i] |= _gpio.R1Mask; _colorsMake[i + 8] |= _gpio.R2Mask; }
+                if ((i & 2) != 0) { _colorsMake[i] |= _gpio.G1Mask; _colorsMake[i + 8] |= _gpio.G2Mask; }
+                if ((i & 4) != 0) { _colorsMake[i] |= _gpio.B1Mask; _colorsMake[i + 8] |= _gpio.B2Mask; }
+            }
+
+            _colorsBuffer = new byte[8 * _fullChainWidth * (_deviceRows >> 1)];
+            _colorsBackBuffer = new byte[8 * _fullChainWidth * (_deviceRows >> 1)];
+
+            _width = width;
+            _rows = height;
+
+            _safeToDispose = true;
+            _swapRequested = false;
+            _startMeasureFrameTime = true;
+        }
+
+        /// <summary>
+        /// Return the width in pixels of the display
+        /// </summary>
+        public int Width => _width;
+
+        /// <summary>
+        /// Return the height in pixels of the display
+        /// </summary>
+        public int Height => _rows;
+
+        /// <summary>
+        /// The height in pixels of one matrix in the chain
+        /// </summary>
+        public int MatrixHeight => _deviceRows;
+
+        /// <summary>
+        /// Set or get the time duration in nanoseconds used in the Pulse Width Modulation (PWM).
+        /// </summary>
+        /// <value></value>
+        public long PWMDuration
+        {
+            get => (long) (((double) _duration / Stopwatch.Frequency) * 1E9);
+            set
+            {
+                _duration = (long) ((double) Stopwatch.Frequency * value / 1E9); // value nanoseconds;
+                _startMeasureFrameTime = true;
+            }
+        }
+
+        /// <summary>
+        /// Return the time in microseconds used to draw a full display frame
+        /// </summary>
+        /// <value></value>
+        public long FrameTime { get => _frameTime; }
```

Comment:
nit: I think this could be just `FrameTime => _frameTime`

Reply:
I'll do that